plugins/experimental/stale_while_revalidate/stale_while_revalidate.c

```diff
@@ -311,46 +324,65 @@ consume_resource(TSCont cont, TSEvent event ATS_UNUSED, void *edata ATS_UNUSED)
     avail = TSIOBufferReaderAvail(state->resp_io_buf_reader);
     TSIOBufferReaderConsume(state->resp_io_buf_reader, avail);
     TSVIONDoneSet(state->r_vio, TSVIONDoneGet(state->r_vio) + avail);
+
```

Comment:
Free all these buffers as soon as they are done, allowing use of "return 0" downcode.